.github/workflows/tests_terraform_examples.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           echo "folders=$folders" >> $GITHUB_OUTPUT
 
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Nitpick - but can we keep this as `latest`? Just to prevent future maintenance :slightly_smiling_face:

Reply:
If use the latest label, there is a risk that tests will suddenly fail one day due to compatibility issues.
That's why I explicitly specified runner version in this PR.

However, if compatibility is maintained when upgrading Runner, as you say, unnecessary maintenance will occur when upgrading.

Both approaches have advantages and disadvantages, but I think it's better to follow a unified policy across the project, so we I modified it to use latest.

e93f3f2325364dc52fbd05527c7a9715d480973b

Reply:
Thank you!